core/src/messages/state.rs

```diff
@@ -11,7 +11,6 @@ use std::net::IpAddr;
 pub struct WorldStateMessage {
     pub cluster: ClusterName,
 
-    #[serde(flatten)]
```

Comment:
needed for u128 ids

Reply:
you are correct, didn't think about that.

that being said though, I think if we're going for no collisions among active locks 64 bit is good..

I wanted to be a bit safer and try for no collisions among any locks, and there it's not so good, since with like 10bn locks there's ~93% chance of collision. https://www.wolframalpha.com/input?i=1+-+%28%282%5E%2864%29-1%29%2F2%5E64%29%5E%28%281e10%29+choose+2%29

but that being said this is only important for breaking the race condition, so u64's should work just fine.

yep will change.